### ISSUE TITLE
Configuration entry duplicated if GitGuardian instance has a trailing slash

### ISSUE THREAD

**Issue by nathaner** [GitGuardian/ggshield#357]
**GitGuardian Shield Version**

- [x] I can reproduce this bug in the latest version

**Command executed**

`GITGUARDIAN_INSTANCE=https://dashboard.gitguardian.com ggshield auth login`

`GITGUARDIAN_INSTANCE=https://dashboard.gitguardian.com/ ggshield auth login`

**Describe the bug**

When targeting a specific GitGuardian instance, the configuration is duplicated if we append a trailing slash to the instance URL.

Here's the configuration displayed by `ggshield config list`:

```
[https://dashboard.gitguardian.com]
default_token_lifetime: None
workspace_id: *****
url: https://dashboard.gitguardian.com
token: *****
token_name: ggshield token 2022-09-14
expiry: never

[https://dashboard.gitguardian.com/]
default_token_lifetime: None
workspace_id: *****
url: https://dashboard.gitguardian.com/
token: *****
token_name: ggshield token 2022-09-14
expiry: never
```

**Expected behavior**

We should only have one configuration entry per instance, wether we append a trailing slash or not.

If we're already signed in on a GitGuardian instance, we should always have the message
`ggshield is already authenticated without an expiry date`
even if we append a trailing slash.

**Screenshots**

N/A

**Traceback (if available)**

N/A

**Comment by Kariss83** [comment]
Hi,
I'm interested in proposing a PR for this issue. I think I found a solution that solves this issue.
Can I open the PR ?

**Comment by agateau-gg** [comment]
Sure, go ahead!

**Comment by agateau-gg** [comment]
Fixed by #366
